specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -29,10 +24,24 @@ import "@azure-tools/typespec-azure-core";
     endpoint: url,
   }
 )
-
 @doc("Azure Messaging EventGrid Client")
 @versioned(ServiceApiVersions)
 namespace Azure.Messaging.EventGrid {
+
```

Comment:
@timotheeguerin: What caused all these comments to move?

Reply:
did those get duplicated?

Reply:
oh I see there were all moved to the top, yeah thats a bug. Comments are always having weird formatting issue

Reply:
filed https://github.com/Azure/typespec-azure/issues/3183

Reply:
If this is a bug do we need to revert the formatting check (and the changes to this file) until this is fixed?